### PR TITLE
update Private Network Access blog post

### PR DESCRIPTION
**Pull request by iVanlIsh**
Changes proposed in this pull request:

- Deprecation trial for non-secure context has been extended to M113.

### CODE REVIEW

File: site/en/blog/private-network-access-update/index.md [hunk 1]
```diff
@@ -75,8 +78,10 @@ strategies:
   for the deprecation trial.
 * September 2021: Chrome 94 rolls out to Stable. Web developers should have signed
   up for the deprecation trial and deployed trial tokens to production.
-* December 2022: Chrome 109 rolls out to Beta.
-* February 2023: Chrome 109 rolls out to Stable. The deprecation trial ends. Chrome
+* December 2022: Origin trial survey sent, feedback received. Deprecation trial
```
Comment (heyawhite) [inline site/en/blog/private-network-access-update/index.md:81]:
```suggestion
* December 2022: Origin trial survey sent and feedback received. The deprecation trial
```

File: site/en/blog/private-network-access-update/index.md [hunk 2]
```diff
@@ -75,8 +78,10 @@ strategies:
...
+  decided to be extended to Chrome 113.
```
Comment (heyawhite) [inline site/en/blog/private-network-access-update/index.md:82]:
```suggestion
  has been extended to Chrome 113.
```

**Review by heyawhite — APPROVED** [review]
Approving these copy changes, but this extension probably warrants an additional short blog post which notes the extension, as readers are not notified of blog post updates.

A new blog post can be just a few paragraphs about the extension, what the feedback was and what's changed, with a link to read more in the previous blog post

**Comment by heyawhite** [comment]
@iVanlIsh we are having that problem across our PRs so I'm not sur. @devnook or @rachelandrew has there been an update on the staging failure?

File: site/en/blog/private-network-access-update-2023-02-02/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+date: 2023-02-02
+hero: 'image/YLflGBAPWecgtKJLqCJHSzHqe2J2/dwtN0NkxkBmIz1EyhzAm.jpg'
+alt: 'Photo by Sara Cohen on Unsplash'
+tags:
+  - chrome-113
+  - security
+---
+
+Due to massive deprecation trial feedbacks mentioning the difficulty of deprecating HTTP
+private webpages and migrating those with TSL certification, we are now announcing an extension of the trial period until Chrome 113. Chrome 113 will roll out to Beta in April 2023 and Stable in May 2023.
```
Comment (letitz) [inline site/en/blog/private-network-access-update-2023-02-02/index.md:16]:
```suggestion
Feedback from websites currently participating in the [Private Network Access from non-secure contexts deprecation trial](https://developer.chrome.com/origintrials/#/view_trial/4081387162304512001) has emphasized the difficulty in migrating affected websites to HTTPS. As a result, the trial is extended until Chrome 113 inclusive. Chrome 114, the first milestone without support for the trial, will roll out to Beta in April 2023 and Stable in May 2023.
```

File: site/en/blog/private-network-access-update-2023-02-02/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (letitz) [inline site/en/blog/private-network-access-update-2023-02-02/index.md:17]: It seems to me that we should mention that we're building a prompt to help, even if we do not give all the details just yet. We can give an overview, point to relevant docs for the motivated reader, and say that we will put out a blog post with more details once it's ready?

File: site/en/blog/private-network-access-update-2023-02-02/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+For further information, please follow [Private Network Access update: Introducing a deprecation trial](/blog/private-network-access-update/).
```
Comment (letitz) [inline site/en/blog/private-network-access-update-2023-02-02/index.md:18]:
```suggestion
For further information, please see [Private Network Access update: Introducing a deprecation trial](/blog/private-network-access-update/). 
```

File: site/en/blog/private-network-access-update-2023-02-02/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+If you are hosting a website within a private network that expects requests from
```
Comment (letitz) [inline site/en/blog/private-network-access-update-2023-02-02/index.md:20]:
```suggestion
If you host a website within a private network that expects requests from
```

File: site/en/blog/private-network-access-update-2023-02-02/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+public networks, we are interested in your feedback and use cases.
+Let us know by filing an issue with Chromium at [crbug.com](crbug.com) and set
+the component to `Blink>SecurityFeature>CORS>PrivateNetworkAccess` or open an issue
+under [Private Network Access WICG specification Github repository](https://github.com/WICG/local-network-access/issues).
```
Comment (letitz) [inline site/en/blog/private-network-access-update-2023-02-02/index.md:24]:
```suggestion
in the [Private Network Access WICG specification Github repository](https://github.com/WICG/local-network-access/issues).
```

File: site/en/blog/private-network-access-update-2023-02-02/index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+Thank you for all the effort you put with us to make the web safer!
```
Comment (letitz) [inline site/en/blog/private-network-access-update-2023-02-02/index.md:26]:
```suggestion
Thank you for helping us make the web safer!
```

**Review by letitz — COMMENTED** [review]
Thanks for writing this up! I've got a couple comments below, mostly rewording.

File: site/en/blog/private-network-access-update-2023-02-02/index.md (hunk 4 above)
Comment (iVanlIsh) [reply site/en/blog/private-network-access-update-2023-02-02/index.md:16]: Done.

**Comment by heyawhite** [comment]
I'll merge this update, the staging error shouldn't be a problem.

**Pull request merged**